sdk/core/core-http/review/core-http.api.md

```diff
@@ -559,7 +559,7 @@ export function promiseToServiceCallback<T>(promise: Promise<HttpOperationRespon
 export type ProxyOptions = ProxySettings;
 
 // @public (undocumented)
-export function proxyPolicy(proxySettings?: ProxySettings): RequestPolicyFactory;
+export function proxyPolicy(proxySettings?: ProxySettings, customNoProxyList?: string[]): RequestPolicyFactory;
```

Comment:
can we put the `noProxyList` onto `ProxySettings` instead? 

If this doesn't make sense, can we wrap it inside of an options object so we don't end up with too many positional parameters?

Reply:
I thought about this too. Putting it into ProxySettings doesn't make a lot of sense to me as we attach it to the request.  I think optional bag is a good idea, assuming we still keep `proxySettings` as is to avoid breaking change.